dataland-frontend/src/components/forms/parts/kpiSelection/euTaxonomyKPIsModel.ts

```diff
@@ -0,0 +1,254 @@
+export const euTaxonomyKPIsModel = {
```

Comment:
Could we call this file something like:

"EuTaxonomyPseudoModelAndMappings"

?

Because:
1.) It is not the actual model here, right. It is just a representation of it.
2.) This file also contains the mappings
3.) All .ts files should start with capital characters.

Furthermore we might want to autogenerate our pseudo models in the future maybe. That way, changes to the actual model would lead to respective changes in the pseudo-model, right?
But I think it is ok for now.

Reply:
yes, right

Reply:
I will leave it like YesNoComponent